ros/src/computing/perception/localization/packages/gpsins_localizer/package.xml

```diff
@@ -0,0 +1,23 @@
+<package format="2">
+    <name>gpsins_localizer</name>
+    <version>0.0.1</version>
+    <description>Localization module based on GPS-INS data</description>
+    <author email="icolwell@autonomoustuff.com">Ian Colwell</author>
+    <maintainer email="icolwell@autonomoustuff.com">Ian Colwell</maintainer>
+    <license>TODO</license>
+
+    <buildtool_depend>catkin</buildtool_depend>
+    <depend>roscpp</depend>
+    <depend>roslib</depend>
+    <depend>nodelet</depend>
+    <depend>tf2</depend>
+    <depend>tf2_ros</depend>
+    <depend>novatel_gps_msgs</depend>
+    <depend>sensor_msgs</depend>
+    <depend>geometry_msgs</depend>
+    <depend>roslint</depend>
```

Comment:
CI is failing due to a missing `<depend>geographiclib</depend>` here for the system dependency for GeographicLib.

Reply:
I didn't know rosdep had a mapping for geographiclib, neat.

Reply:
Check out http://rosindex.github.io/deps/ for system dependencies. These and all of the "package_name" -> UbuntuPackageName translations are at https://github.com/ros/rosdep.

Reply:
Also - I didn't know this but since we only have `master` and PRs turned on for CI, if you make changes on your PR, you have to kick CI to get it to run again. I just got it going to re-test this PR.

Reply:
That's strange, if "build PRs" is enabled, usually any new commit to the PR will also build.
I think it might fail though, I was using geographic lib installed from source. I'll see if what changes are need to get it to build from the apt version.

Reply:
So it turns out the apt-installed geographic lib has a broken cmake file (https://stackoverflow.com/questions/48169653/finding-geographiclib-in-cmake-on-debian/48524106#48524106).
I was able to get it to build by linking directly to it, which is not ideal, but works for now.

Reply:
Yeah, I found that too. I used the answerer's FindGeographicLib.cmake solution and it worked as well. Do you want me to push that version so the environment doesn't have to be modified?

Reply:
What environment modification are you referring to? All I did was change the linking.

Reply:
Oh, I gotcha. One of the answers on that SO question said to create a symbolic link. I thought that was the linking you were talking about. The only downside to your version is that CMake won't warn you ahead of compilation. Not a big deal, though.